scim2-sdk-common/src/main/java/com/unboundid/scim2/common/filters/OrFilter.java

```diff
@@ -22,7 +22,32 @@
 import java.util.List;
 
 /**
- * Logical OR combining filter.
+ * This class represents a SCIM {@code or} filter. An OR filter allows
+ * a client to specify two different filter criteria where at least one
+ * of the criterion must match. For instance, consider the following filter.
```

Comment:
Should be either "at least one criterion must match" or "at least one of the criteria must match".